qcommon/test/org/qcmg/common/vcf/VcfUtilsTest.java

```diff
@@ -52,45 +52,6 @@ public class VcfUtilsTest {
 	}
 	
 	
-	@Test
-	public void getAltFrequencyTest() throws Exception{
```

Comment:
why delete getAltFrequencyTest()?  you can move it into VcfFormatFieldRecordTest if you think it is not related to VcfUtils.